decidim-elections/app/views/decidim/votings/votings/_check_fields.html.erb

```diff
@@ -12,9 +12,9 @@ form.options_for_document_type_select,
 
   <div class="flex gap-2">
 
-    <%= f.text_field :day, placeholder: t(".day_placeholder"), label: t(".day"), "data-autojump" => true, "data-max-length" => 2, "data-jump-next" => "#login_month" %>
-    <%= f.text_field :month, placeholder: t(".month_placeholder"), label: t(".month"), "data-autojump" => true, "data-max-length" => 2, "data-jump-prev" => "#login_day", "data-jump-next" => "#login_year" %>
-    <%= f.text_field :year, placeholder: t(".year_placeholder"), label: t(".year"), "data-autojump" => true, "data-max-length" => 4, "data-jump-prev" => "#login_month" %>
+    <%= f.text_field :day, placeholder: t(".day_placeholder"), label: t(".day"), "data-autojump" => true, "data-max-length" => 2, "data-jump-next" => "#check_month" %>
+    <%= f.text_field :month, placeholder: t(".month_placeholder"), label: t(".month"), "data-autojump" => true, "data-max-length" => 2, "data-jump-prev" => "#check_day", "data-jump-next" => "#check_year" %>
+    <%= f.text_field :year, placeholder: t(".year_placeholder"), label: t(".year"), "data-autojump" => true, "data-max-length" => 4, "data-jump-prev" => "#check_month" %>
```

Comment:
```suggestion
    <%= f.text_field :day, placeholder: t(".day_placeholder"), label: t(".day"), "data-autojump" => true, "data-max-length" => 2, "data-jump-next" => "##{f.object_name}_month" %>
    <%= f.text_field :month, placeholder: t(".month_placeholder"), label: t(".month"), "data-autojump" => true, "data-max-length" => 2, "data-jump-prev" => "##{f.object_name}_day", "data-jump-next" => "##{f.object_name}_year" %>
    <%= f.text_field :year, placeholder: t(".year_placeholder"), label: t(".year"), "data-autojump" => true, "data-max-length" => 4, "data-jump-prev" => "##{f.object_name}_month" %>
```

Reply:
The above code seems to fix the issue on the frontend side ... i have not checked the tests.. .